docs/reference/src/documentation/introduction/installation/index.md

```diff
@@ -18,9 +18,9 @@ The supported operating systems include Linux and macOS; however, Windows is [`u
 
 `Cargo` may be used instead of [`Fuelup`](fuelup.md); however, the user needs to manage the toolchain themselves.
 
-The advantage of using `Cargo` is the installation of [`plugins`](../../forc/plugins/index.md) that have not been added into [`Fuelup`](fuelup.md).
+The advantage of using `Cargo` is the installation of [`plugins`](https://fuellabs.github.io/sway/master/book/forc/plugins/index.html) that have not been added into [`Fuelup`](fuelup.md).
```

Comment:
It seems to me we should either use the FQDN path or use relative paths linking to .md files everywhere for consistency.

Reply:
see comment above